README.md

```diff
@@ -9,25 +9,31 @@ with [VSCode](https://github.com/neoclide/coc.nvim).
 
 ### Normal Installation
 
+Install from Marketplace or install manually.
+
 1. Download the `phpactor.vsix` file from the [lastest release](https://github.com/phpactor/vscode-phpactor/releases/latest)
 2. Run `code --install-extension /path/to/phpactor.vsix`
 
-### Development Installation
+### Development
 
 For development it is easier
 
 1. Install [npm](https://www.npmjs.com/get-npm).
 2. Install typescript: `npm install -g typescript`
 3. Git clone this package
 4. `npm install`
-5. `cd ~/.vscode/extensions`. If running [VS Code server on WSL](https://code.visualstudio.com/docs/remote/wsl): `cd ~/.vscode-server/extensions`
-6. `ln -s /path/to/vscode-phpactor`
+5. Run `composer install`
+6. Open the folder in VSCode
+7. Start watch compilation by selecting `Terminal / Run Build Task...`
+8. Open the Run and Debug side menu, select `Launch Extension` from the debug configuration and hit run (`F5`)
+9. Additionally you can also run `Listen for Xdebug` to debug the Language Server - but has to be run before the Language Server starts.
```

Comment:
no need to change this, but JFYI instead of numbers you can use `.` and it numbers them automatically IIRC :)